classes/import-export.php

```diff
@@ -501,7 +501,7 @@ public function import_csv() {
 						echo '<h2>' . sprintf( __( 'View them <a href="%s">here</a>', 'SimpleMap' ), admin_url( 'edit.php?post_type=sm-location' ) ) . '</h2>';
 					}
 
-					// Import is finished, delete csv and redirect to edit locaitons page.
+					// Import is finished, delete csv and redirect to edit locations page.
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->